app/controllers/users_controller.rb

```diff
@@ -1,7 +1,66 @@
 class UsersController < ApplicationController
 
-  def list
+  def show
     @users = User.all
   end
-  alias :index :list
+  alias :index :show
+
+  def new
+  end
+
+  def create
+    @user = User.create(user_params)
+    redirect_to '/'
+  end
+
+  def trim
+    user = User.find(params[:id])
```

Comment:
All 3 below methods basically do all the setup the same, do one thing different, and then render the same.  While functionally they are great, your code could be more DRY (http://en.wikipedia.org/wiki/Don%27t_repeat_yourself).  By using before_filter, after_filter, or around_filters.
See: http://guides.rubyonrails.org/action_controller_overview.html#filters